scripts/lint/ourlint.js

```diff
@@ -199,6 +205,37 @@ const TESTS = {
       return `Invalid URLs:\n\n${invalidUrls.join('\n\n')}`;
     }
   },
+
+  /* Check if all licenses are valid SPDX identifiers */
+  async checkLicense(data) {
+    const spdxLicenseIds = JSON.parse(
+      await fs.readFile(
+        path.join(
+          getDirnameFromImportMeta(import.meta.url),
+          '..',
+          '..',
+          'node_modules/spdx-license-ids/index.json',
+        ),
+        'utf8',
+      ),
+    );
+    const badLicenses = [];
+    for (const {title, slug, license} of data.icons) {
+      if (
+        license &&
+        license.type !== 'custom' &&
+        !spdxLicenseIds.includes(license.type)
+      ) {
+        badLicenses.push(
+          `${title} (${slug ?? titleToSlug(title)}) has an bad license: ${license.type}`,
```

Comment:
I think that the message is not very descriptive. Would expect something like `{title} does not have a valid SPDX license. See the valid licenses at {spdxLicensesUrl}.`

Reply:
Good point.